src/transformers/models/bit/modeling_bit.py

```diff
@@ -842,7 +842,7 @@ def forward(
     """,
     BIT_START_DOCSTRING,
 )
-class BitBackbone(BitPreTrainedModel):
+class BitBackbone(BitPreTrainedModel, BackboneBaseModel):
```

Comment:
It's not really convenient that any backbone will need to inherit from 2 classes can we directly incorporate this into PreTrainedModel or not?

Reply:
SGTM, but just let @sgugger to confirm

Reply:
I don't see where the problem lies. That's the most Pythonic way of doing this (compared to having all models inherit from `GenerationMixin` for instance, whereas lots of them shouldn't). You give the type to objects that need it (and only those).